src/express-route-cross-connection/azext_expressroutecrossconnection/sdk/__init__.py

```diff
@@ -0,0 +1,17 @@
+﻿# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from .network_management_client import NetworkManagementClient
+from .version import VERSION
+
+__all__ = ['NetworkManagementClient']
+
+__version__ = VERSION
```

Comment:
Please put the autogenerated sdks being vendored under a folder named `vendored_sdks`, this will stop static checking of those files.

Reply:
This did not stop static checking of the files.  Static check failed because files under `vendored_sdks` didn't have license headers...